project/gh-action/infrastructure/kubernetes/configmap.yaml

```
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: web-config
+data:
+  DB_PASSWORD: "sensitive-data-in-configmap"  
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)